ui/src/app/tasks/task-details/task-details.component.html

```diff
@@ -36,12 +36,20 @@
       <td>Batch Job</td>
       <td>
         <span *ngIf="taskExecution.jobExecutionIds.length > 0" class="glyphicon glyphicon-ok text-info"></span>
-        <span *ngIf="taskExecution.jobExecutionIds.length == 0" class="glyphicon glyphicon-remove"></span>
+        <span *ngIf="taskExecution.jobExecutionIds.length == 0" class="glyphicon glyphicon-remove text-danger"></span>
       </td>
     </tr>
     <tr>
       <td>Job Execution Ids</td>
-      <td></td>
+      <td *ngIf="taskExecution.jobExecutionIds.length > 0" id="jobExecutionIds">
+        <button class="btn btn-default" type="button" (click)="goToJobExecution(jobExecutionId)"
+          *ngFor="let jobExecutionId of taskExecution.jobExecutionIds">
+          {{ jobExecutionId }}
+        </button>
+      </td>
+      <td *ngIf="taskExecution.jobExecutionIds.length == 0">
+          N/A
```

Comment:
Should we omit the N/A and just leave the field blank?

Reply:
Nevermind.  I looked for an example where we use N/A in other parts of the UI project and we do.   LGTM.  I'll fix the misspell on merge